lms/djangoapps/edxnotes/tests.py

```diff
@@ -525,43 +525,6 @@ def test_search_empty_collection(self, mock_get):
             json.loads(helpers.get_notes(self.request, self.course))
         )
 
-    def test_preprocess_collection_escaping(self):
```

Comment:
I would have expected some test to change that verifies that a tag is highlighted as expected.  I don't see anything like that.